Source/Diffusion.cpp

```diff
@@ -126,6 +126,8 @@ PeleC::getMOLSrcTerm(
       const amrex::Box cbox = amrex::grow(vbox, ng - 1);
       auto const& MOLSrc = MOLSrcTerm.array(mfi);
 
+      const int captured_clean_massfrac = clean_massfrac;
```

Comment:
Can we move all these captures to just before the `ParallelFor` in which they are used?

Reply:
Its done now except in react where it is used in multiple places